atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsAssert.java

```diff
@@ -139,18 +165,37 @@ public void hasNotRegisteredTargetedOutcome(SweepStrategy strategy, SweepOutcome
                         AtlasDbMetricNames.TAG_STRATEGY, getTagForStrategy(strategy)));
     }
 
+    private Double getEntriesReadInBatchMeanConservative() {
+        return (Double) getGaugeConservative(AtlasDbMetricNames.BATCH_SIZE_MEAN).getValue();
+    }
+
+    private Double getEntriesReadInBatchMeanThorough() {
+        return (Double) getGaugeThorough(AtlasDbMetricNames.BATCH_SIZE_MEAN).getValue();
```

Comment:
yeah, seems we still need to cast so the params above don't seem to be helpful, unless I'm missing something